HW0/P5/P5.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
```

Comment:
From your P5 graph, it is hard to tell that the multiple workers is logarithmic.  It is also unlabeled, which makes the graph hard to interpret.